extensions/2.0/Vendor/EXT_feature_metadata/README.md

```diff
@@ -201,7 +180,9 @@ Often per-texel feature IDs provide finer granularity than per-vertex feature ID
 }
 ```
 
-`texture` is a glTF [`textureInfo`](../../../../../specification/2.0/schema/textureInfo.schema.json) object. `channels` must be a single channel (`"r"`, `"g"`, `"b"`, or `"a"`). Furthermore, feature IDs must be whole numbers in the range `[0, count - 1]` (inclusive), where `count` is the total number of features in the feature table. Texture filtering should be disabled when accessing feature IDs.
+The `featureId` entry for a feature ID texture extends the glTF [`textureInfo`](../../../../../specification/2.0/schema/textureInfo.schema.json) object. Each `channel` must be a positive integer corresponding to a channel of the source texture. Channels of an `RGBA` texture are numbered 0–3 respectively, although specialized texture formats may allow additional channels. Feature IDs must be whole numbers — stored in linear space — in the range `[0, count - 1]` (inclusive), where `count` is the total number of features in the feature table.
```

Comment:
The part about specialized texture formats... it feels rather vague to just say "may allow additional channels".

For other texture formats (e.g. GeoTIFF), would it be safe to say that the channel indices must match the order of the channels in the file? Or are there any image formats that use an unordered dictionary of `channel -> data`?

Reply:
For GeoTIFF (and [Layered TIFF](https://wiki.panotools.org/TIFF#Layers)), layers or bands do have a well-defined order. I'm not aware of any image formats in which the channels don't have a meaningful order.

In KTX2 (`.ktx2`) or Basis (`.basis`), or other compressed texture formats, it may be possible to store an array texture in a single file to provide more than 3-4 total channels. In order to support those we would probably need a two-level indexing system, e.g. `{layerIndex: 3, channel: 1}`.

Reply:
Reading through this again on a new day, I think this is fine for now since we don't have any specific requirements of supporting other texture formats right now.